kong/templates/nginx_kong.lua

```diff
@@ -234,7 +234,7 @@ server {
     location / {
         default_type application/json;
         content_by_lua_block {
-            Kong.serve_admin_api()
```

Comment:
Does this introduce a breaking change to anyone running Kong with a custom Nginx template? If so, I'd rather avoid the style change now (and possibly postpone it to 2.0).

Looking at the other commit it seems that if they fail to update the template they just don't get the latency header, which is harmless, but with this one they're forced to update the template otherwise the Admin API doesn't work.

Alternatively, we could add `kong.serve_admin_api = kong.admin_content` in the Kong code to make it backwards compatible?

Reply:
Ok, I will make it an alias.

Reply:
Alias made.